third_party/2and3/bleach/sanitizer.pyi

```diff
@@ -1,7 +1,7 @@
-from typing import List, Dict, Any, Optional, Type, Pattern, Union, Callable, Container, Text
+from typing import Any, Callable, Container, Dict, Iterable, List, Optional, Pattern, Text, Type, Union
 
 ALLOWED_TAGS: List[Text]
-ALLOWED_ATTRIBUTES: Dict[Text, List[Text]]
+ALLOWED_ATTRIBUTES: Dict[Text, Container[Text]]
```

Comment:
Why did you change this type? All the other constants use concrete types and this seems more appropriate here as well.

Reply:
Oh, I'm not sure. You're right, that should probably stay as `List`. I pushed another commit reverting that now.

Do you want me to squash these into a single commit?

Reply:
Thanks! We prefer not squashing for easier reviewability. We will squash when merging.